pkgs/data/fonts/material-design-icons/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "material-design-icons-${version}";
+  version = "3.2.89";
+
+  src = fetchFromGitHub {
+    owner  = "Templarian";
+    repo   = "MaterialDesign-Webfont";
+    rev    = "v${version}";
+    sha256 = "1rxaiiij96kqncsrlkyp109m36v28cgxild7z04k4jh79fvmhjvn";
+  };
+
+  buildCommand = ''
+    mkdir -p $out/share/fonts/{eot,svg,truetype,woff,woff2}
+    cp $src/fonts/*.eot $out/share/fonts/eot/
```

Comment:
`$src/` is not necessary -- the source is unpacked in the current working directory.